src/go/k8s/apis/redpanda/v1alpha1/console_types.go

```diff
@@ -39,6 +39,18 @@ type ConsoleSpec struct {
 
 	Deployment Deployment `json:"deployment"`
 	Connect    Connect    `json:"connect"`
+
+	Enterprise *Enterprise `json:"enterprise,omitempty"`
```

Comment:
I think this is better handled by Console as the installation should I think still work even if you don't provide a license

Reply:
I think the UX is the same with installing Console without the operator. If user adds Enterprise features without specifying a valid license, Console will still run and just ignores the Enterprise fields. In my opinion, the operator should only validate fields that will make the underlying Console pods crashloopbackoff.

Reply:
The operator could be much more tight to software that it's manage. I don't see this operator as generic controller. If we could on the kubectl apply verify that something is miss configured then it would easily inform with nice error message.